variables.tf

```diff
@@ -145,3 +145,19 @@ variable "sql_memory_events_alert_policy_threshold_duration" {
   type    = string
   default = "900s"
 }
+
+# read replica
+variable "create_read_replica" {
```

Comment:
Do we need this one?
Based on the docs if we pass the master name it will automatically become a replica and if not will be a master itself.

Reply:
it is mainly used for the replica_configuration block that needs to be changed conditionally for replica otherwise will trigger a recreation of the actual instances that we have, and I've added everywhere where the read_replica it's involved for better visibility